logConfig.yaml

```diff
@@ -0,0 +1,32 @@
+schedTrace:
+  enabled: true
+  filenameExtension: _schedTrace.log
+  allowOnConsole: true
+sps:
+  enabled: false
+  filenameExtension: _schedPolicySwitch.log
+  allowOnConsole: true
+console:
+  enabled: true
+  filenameExtension: _console.log
+  minLogLevel: debug
+  allowOnConsole: true
+pcp:
+  enabled: true
+  filenameExtension: .pcplog
+  allowOnConsole: false
+schedWindow:
+  enabled: true
```

Comment:
disable schedWindow logging in the log config file that is kept within the repository.

Reply:
Done